packages/components/src/reaction_selection_menu/utils.jsx

```diff
@@ -0,0 +1,27 @@
+import { position, screenPosition } from 'nebenan-helpers/lib/dom';
+
+const VERTICAL_THRESHOLD = 30;
+export const getItemIndexForPosition = (positions, { top, height }, { x, y }) => {
+  const isAboveElement = y < top - VERTICAL_THRESHOLD;
+  const isBelowElement = y > top + height + VERTICAL_THRESHOLD;
+  if (isAboveElement || isBelowElement) return null;
+
+  let touchedIndex = 0;
+  for (let i = positions.length - 1; i >= 0; i -= 1) {
```

Comment:
can't we get away with touchenter/touchleave on nodes?

Reply:
I think this was one of my first approaches but it failed. Don't remember why. Will try it when I have some time left.

Reply:
worth investigating imo, this is not ideal

Reply:
`touchenter` and `touchleave` events don't exist. we only have `touchstart`, `touchend`, `touchcancel` and `touchmove`

As far as I understand, `touchmove` only gets triggered on the element that received `touchstart`. That means that I cannot use `touchmove` on reaction list items because the initial touch started on a parent

Reply:
Played around with custom events and `elementFromPoint` but it is even more complicated than the current approach. Would leave it as it is.